source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,135 @@
+
+After installing the driver and OVN, the OVN-controller must be configured. This can be done either using the vdsm-tool:
+
+    `vdsm-tool ovn-config <OVN central IP> <local IP used for OVN tunneling>`
+
+or by using the OVN command-line interface directly. For more information about OVN-controller setup, please check the
+[OVS ducumentation](http://openvswitch.org/support/dist-docs/) 
+
+## oVirt OVN Provider
+
+The oVirt OVN provider is a proxy that the oVirt Engine uses to interact with OVN. It is delivered as an rpm that is to be installed on the host where OVN central is installed.
+
+The [oVirt OVN provider RPM](http://resources.ovirt.org/repos/mmirecki/ovirt-provider-ovn-0-1.noarch.rpm) is also available now. The latest version can always be downloaded and built from the repository as well. Once the rpm is downloaded, it can be installed with this command:
+
+    `yum install ovirt-provider-ovn-0-1.noarch.rpm`
+
+OVN requires OVN to be installed on the host (version 2.6 or later). The following OVN packages must be installed:
+
+* `openvswitch`
+* `openvswitch-ovn-common`
+* `openvswitch-ovn-central`
+* `python-openvswitch`
+
+These are also available from the [OVS website](http://openvswitch.org/download/) or built using the code downloaded from the OVS repo (described below).
+
+After installing the oVirt OVN provider, the admin needs to open up port 9696 in the firewall.
+This can be done manually or by adding the ovirt-provider-ovn firewalld service to the appropriate firewalld zone.
+
+`firewall-cmd --zone=<zone to add service to> --add-service=ovirt-provider-ovn --permanent`
+`firewall-cmd --reload`
+
+The zone currently selected as default can be obtained by executing:
+
+`firewall-cmd --get-default-zone`
+
+After installation, the provider can be started as follows:
+
+	`systemctl start ovirt-provider-ovn`
+
+The provider can then be added to oVirt as an external network provider. In order to add a new provider, go to the External Providers section in the oVirt UI and click the Add button.
+
+![adding a new provider](uarwo40-edit-cluster.png)
+
+When the provider is successfully added, existing OVN networks can be imported to oVirt.
+New OVN networks can be defined using oVirt by adding a network and specifying it to be added on an external provider (make sure you add the external provider in read-write mode, so that you can add external networks from oVirt).
+
+A VM NIC can be added to OVN networks by simply choosing an external network during NIC provisioning.
+
+OVN based networking brings many advantages to oVirt:
+* More granular security - it brings complete network isolation without the need for defining VLANs.
+* Less resources - since the OVN networks are logical overlays, many of them can be defined on the same underlying physical infrastructure. As oposed to traditional oVirt networking, where each network required a separate host NIC, all the OVN networks can be acommodated on a single NIC.
```

Comment:
but I'd mention the CPU cost of using geneve tunnelling.

As oposed (sic) to end: that's not correct, due to VLANs. So let us drop this sentence.